/home/aih/sebastian.birk/miniconda3/envs/nichecompass/lib/python3.9/site-packages/umap/distances.py:1063: NumbaDeprecationWarning: [1mThe 'nopython' keyword argument was not supplied to the 'numba.jit' decorator. The implicit default value for this argument is currently False, but it will be changed to True in Numba 0.59.0. See https://numba.readthedocs.io/en/stable/reference/deprecation.html#deprecation-of-object-mode-fall-back-behaviour-when-using-jit for details.[0m
  @numba.jit()
/home/aih/sebastian.birk/miniconda3/envs/nichecompass/lib/python3.9/site-packages/umap/distances.py:1071: NumbaDeprecationWarning: [1mThe 'nopython' keyword argument was not supplied to the 'numba.jit' decorator. The implicit default value for this argument is currently False, but it will be changed to True in Numba 0.59.0. See https://numba.readthedocs.io/en/stable/reference/deprecation.html#deprecation-of-object-mode-fall-back-behaviour-when-using-jit for details.[0m
  @numba.jit()
/home/aih/sebastian.birk/miniconda3/envs/nichecompass/lib/python3.9/site-packages/umap/distances.py:1086: NumbaDeprecationWarning: [1mThe 'nopython' keyword argument was not supplied to the 'numba.jit' decorator. The implicit default value for this argument is currently False, but it will be changed to True in Numba 0.59.0. See https://numba.readthedocs.io/en/stable/reference/deprecation.html#deprecation-of-object-mode-fall-back-behaviour-when-using-jit for details.[0m
  @numba.jit()
/home/aih/sebastian.birk/miniconda3/envs/nichecompass/lib/python3.9/site-packages/umap/umap_.py:660: NumbaDeprecationWarning: [1mThe 'nopython' keyword argument was not supplied to the 'numba.jit' decorator. The implicit default value for this argument is currently False, but it will be changed to True in Numba 0.59.0. See https://numba.readthedocs.io/en/stable/reference/deprecation.html#deprecation-of-object-mode-fall-back-behaviour-when-using-jit for details.[0m
  @numba.jit()
/home/aih/sebastian.birk/miniconda3/envs/nichecompass/lib/python3.9/site-packages/anndata/_core/anndata.py:117: ImplicitModificationWarning: Transforming to str index.
  warnings.warn("Transforming to str index.", ImplicitModificationWarning)
/home/aih/sebastian.birk/miniconda3/envs/nichecompass/lib/python3.9/site-packages/anndata/_core/anndata.py:117: ImplicitModificationWarning: Transforming to str index.
  warnings.warn("Transforming to str index.", ImplicitModificationWarning)
/home/aih/sebastian.birk/miniconda3/envs/nichecompass/lib/python3.9/site-packages/anndata/_core/anndata.py:117: ImplicitModificationWarning: Transforming to str index.
  warnings.warn("Transforming to str index.", ImplicitModificationWarning)
/home/aih/sebastian.birk/miniconda3/envs/nichecompass/lib/python3.9/site-packages/anndata/_core/anndata.py:117: ImplicitModificationWarning: Transforming to str index.
  warnings.warn("Transforming to str index.", ImplicitModificationWarning)
/home/aih/sebastian.birk/miniconda3/envs/nichecompass/lib/python3.9/site-packages/anndata/_core/anndata.py:1830: UserWarning: Observation names are not unique. To make them unique, call `.obs_names_make_unique`.
  utils.warn_names_duplicates("obs")
Traceback (most recent call last):
  File "/home/aih/sebastian.birk/workspace/projects/nichecompass-reproducibility/scripts/ablation/../train_nichecompass_reference_model.py", line 765, in <module>
    gene_expr_recon_dist=args.gene_expr_recon_dist,
  File "/home/aih/sebastian.birk/workspace/projects/nichecompass/nichecompass/models/nichecompass.py", line 689, in train
    self.trainer.train(
  File "/home/aih/sebastian.birk/workspace/projects/nichecompass/nichecompass/train/trainer.py", line 477, in train
    if self.edge_val_loader is not None:
  File "/home/aih/sebastian.birk/miniconda3/envs/nichecompass/lib/python3.9/site-packages/torch/utils/_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "/home/aih/sebastian.birk/workspace/projects/nichecompass/nichecompass/train/trainer.py", line 655, in eval_end
    edge_incl=edge_incl_val_accumulated,
  File "/home/aih/sebastian.birk/miniconda3/envs/nichecompass/lib/python3.9/site-packages/mlflow/tracking/fluent.py", line 641, in log_metric
    MlflowClient().log_metric(run_id, key, value, int(time.time() * 1000), step or 0)
  File "/home/aih/sebastian.birk/miniconda3/envs/nichecompass/lib/python3.9/site-packages/mlflow/tracking/client.py", line 799, in log_metric
    self._tracking_client.log_metric(run_id, key, value, timestamp, step)
  File "/home/aih/sebastian.birk/miniconda3/envs/nichecompass/lib/python3.9/site-packages/mlflow/tracking/_tracking_service/client.py", line 292, in log_metric
    self.store.log_metric(run_id, metric)
  File "/home/aih/sebastian.birk/miniconda3/envs/nichecompass/lib/python3.9/site-packages/mlflow/store/tracking/file_store.py", line 843, in log_metric
    check_run_is_active(run_info)
  File "/home/aih/sebastian.birk/miniconda3/envs/nichecompass/lib/python3.9/site-packages/mlflow/entities/run_info.py", line 12, in check_run_is_active
    raise MlflowException(
mlflow.exceptions.MlflowException: The run aeb553e41d7f4914aa1b49efb81c5ca8 must be in 'active' lifecycle_stage.
Error in atexit._run_exitfuncs:
Traceback (most recent call last):
  File "/home/aih/sebastian.birk/miniconda3/envs/nichecompass/lib/python3.9/site-packages/mlflow/tracking/fluent.py", line 393, in end_run
    MlflowClient().set_terminated(run.info.run_id, status)
  File "/home/aih/sebastian.birk/miniconda3/envs/nichecompass/lib/python3.9/site-packages/mlflow/tracking/client.py", line 1575, in set_terminated
    self._tracking_client.set_terminated(run_id, status, end_time)
  File "/home/aih/sebastian.birk/miniconda3/envs/nichecompass/lib/python3.9/site-packages/mlflow/tracking/_tracking_service/client.py", line 468, in set_terminated
    self.store.update_run_info(
  File "/home/aih/sebastian.birk/miniconda3/envs/nichecompass/lib/python3.9/site-packages/mlflow/store/tracking/file_store.py", line 549, in update_run_info
    check_run_is_active(run_info)
  File "/home/aih/sebastian.birk/miniconda3/envs/nichecompass/lib/python3.9/site-packages/mlflow/entities/run_info.py", line 12, in check_run_is_active
    raise MlflowException(
mlflow.exceptions.MlflowException: The run aeb553e41d7f4914aa1b49efb81c5ca8 must be in 'active' lifecycle_stage.
